src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -7747,6 +7893,24 @@ private static string GetDatesExpectedString(bool prettyPrint, string keyString,
             return Encoding.UTF8.GetString(ms.ToArray());
         }
 
+        private static void CompensateWhitespaces(bool prettyPrint, JsonTextWriter json, TextWriter streamWriter, int whitespaceCount = 1)
```

Comment:
honestly I'm not sure what's the intent of this method...

Reply:
Previously we may need this helper method to instrument underlying stream to add additional newline and indented whitespaces during call newtonsoft.json methods because newtonsoft.json also did not do what proposal describes.